dashboard/app/views/maker/setup.html.haml

```diff
@@ -1,5 +1,4 @@
-%script{src: minifiable_asset_path('js/makerlab.js')}
```

Comment:
A-ha!  @bcjordan removed the `makerlab.js` entry point [here on January 18th](https://github.com/code-dot-org/code-dot-org/pull/12685/files#diff-26cdc4306e5d33ea4df1299321d8852aL396) but this usage got missed, (my bad, I reviewed that change).